### ISSUE TITLE
[Website] Activity Update: Ensures elements with an ARIA role that require child roles contain them (.ac-actionSet[role="menubar"])

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6425]
### Target Platforms

NodeJS

### SDK Version

https://adaptivecards.io/  9/9/2021

### Application Name

Website

### Problem Description

[ADO 35918589](https://microsoft.visualstudio.com/OS/_workitems/edit/35918589)

**Repro Steps:**
1. Open above URL with valid credential.
2. Navigate to 'Samples' link and press tab key to activate it.
3. Select the 'Activity Update' option and navigate to 'Date Picker' under 'Publish Adaptive Card Schema'.
4. Run Accessibility insight for Web and observe the issue.

**Actual Result:**
Ensures elements with an ARIA role that require child roles contain them (.ac-actionSet[role="menubar"])

**Expected Result:**
Ensures elements should be with an ARIA role that require child roles contain them (.ac-actionSet[role="menubar"])

### Card JSON

```JSON
ActivityUpdate.json
```
